### PR TITLE
lazpaint: 7.2.2 -> 7.2.2-unstable-2024-01-20

### PR DESCRIPTION
**Pull request by superherointj**
lazpaint: 7.2.2 -> 7.2.2-unstable-2024-01-20

Fixes #181093

### CODE REVIEW

File: pkgs/applications/graphics/lazpaint/default.nix [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
     repo = "bgracontrols";
-    rev = "v7.6";
-    sha256 = "sha256-btg9DMdYg+C8h0H7MU+uoo2Kb4OeLHoxFYHAv7LbLBA=";
+    rev = "v8.0";
+    hash = "sha256-5L05eGVN+xncd0/0XLFN6EL2ux4aAOsiU0BMoy0dKgg=";
   };
 in stdenv.mkDerivation rec {
   pname = "lazpaint";
-  version = "7.2.2";
+  version = "unstable-2024-01-20";
```
Comment (h7x4) [inline pkgs/applications/graphics/lazpaint/default.nix:21]: This should be `7.2.2-unstable-2024-01-20` - https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

**Comment by h7x4** [comment]
@ofborg eval

**Comment by superherointj** [comment]
> @ofborg eval

I have rebased just in case. eval locally is fine. And package is building for me.

**Comment by h7x4** [comment]
I think ofborg is just running low on space. Let's wait a little, and retry later

**Comment by superherointj** [comment]
> I think ofborg is just running low on space. Let's wait a little, and retry later

CI is green now. (as expected)

**Review by h7x4 — APPROVED** [review]

**Comment by superherointj** [comment]
Thanks @h7x4 !

**Pull request merged**